packages/components/pages/f-account-info/src/components/AccountInfo.vue

```diff
@@ -1,49 +1,175 @@
 <template>
-    <div
-        :class="$style['c-accountInfo']"
-        data-test-id="accountInfo">
-        {{ copy.text }}
-    </div>
+    <card
+        :card-heading="$t('accountDetails')"
+        data-test-id="account-info-card"
+        has-outline
+        is-page-content-wrapper>
+        <h2
+            class="u-spacingBottom--large">
+            {{ $t('yourDetails') }}
+        </h2>
+
+        <form
+            method="post"
+            @submit.prevent="onFormSubmit">
+            <form-field
+                :label-text="$t('fields.emailAddressLabel')"
+                disabled
+                :placeholder="$t('fields.emailAddressPlaceholder')"
+                class="u-spacingBottom--large"
+                :value="fields.emailAddress" />
+
+            <f-link
+                is-distinct
+                :class="$style['c-accountInfo-customerCareLink']"
+                href="/help"
+                target="_blank">
+                {{ $t('contactCustomerCareTeam') }}
+            </f-link>
+
+            <form-field
+                :label-text="$t('fields.firstNameLabel')"
+                :placeholder="$t('fields.firstNamePlaceholder')"
+                :value="fields.firstName" />
+
+            <form-field
+                :label-text="$t('fields.lastNameLabel')"
+                :placeholder="$t('fields.lastNamePlaceholder')"
+                :value="fields.lastName" />
+
+            <h2
+                class="u-spacingBottom--large">
+                {{ $t('deliveryAddress') }}
+            </h2>
+
+            <form-field
```

Comment:
Doesn't need to be done as part of this PR, but I'd check with @Preet-Sandhu / @ross-gratton if these form-fields could benefit from being grouped into fieldsets, from an accessibility standpoint.

Reply:
Ill ask and pick it up in my next PR if needed